docker/travis_test/entrypoint.sh

```
@@ -0,0 +1,43 @@
+#!/bin/bash                                                                                                                                                                                                        
```

How about `#!/bin/sh -e` instead to not ignore errors?

---

any reason to change shell?

why not just 
> set -e
?

---

I think sticking with bash is fine, too (at least then we know what we get). I usually also use `set -e` separately. Shouldn't make any difference.